modules/oe_content_consultation/config/install/field.field.node.oe_consultation.oe_consultation_deadline.yml

```diff
@@ -0,0 +1,20 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.node.oe_consultation_deadline
+    - node.type.oe_consultation
+  module:
+    - datetime
+id: node.oe_consultation.oe_consultation_deadline
+field_name: oe_consultation_deadline
+entity_type: node
+bundle: oe_consultation
+label: 'Deadline date'
```

Comment:
Based on spec https://webgate.ec.europa.eu/fpfis/wikis/pages/viewpage.action?spaceKey=EWPP&title=Consultation+Content+Type+Specification+-+Version+1.0.0 it has to be "Deadline"